addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -85,6 +85,9 @@
     'SM': {'9951': 'vat'},
     'TR': {'9952': 'vat'},
     'VA': {'9953': 'vat'},
+    'IT': {'0210': 'l10n_it_codice_fiscale'},
+    'DK': {'0198': 'vat'},
+    'IS': {'0196': 'vat'},
```

Comment:
I think we should keep in using the alphabetic order :p (even if it seems already to late for some entries)